pyomo/contrib/pynumero/doc/source/installation.rst

```diff
@@ -0,0 +1,122 @@
+Installation
+======================================
+
+PyNumero is distributed within the python optimization module pyomo.
+
+To build PyNumero from github (currently on a fork)::
+	
+	git clone https://github.com/santiagoropb/pyomo.git
```

Comment:
Will this still be accurate after merging this PR?

Reply:
updated to pyomo repo